TL improvements (#314)
07th-mod/higurashi-console-arcs: Update/omot_011.txt

//r静まり返った中、暗くなった地面を照らし出すダイナモの音が、いやにやかましく響き渡っていた。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "静まり返った中、暗くなった地面を照らし出すダイナモの音が、いやにやかましく響き渡っていた。",

> 			NULL, "The surrounding area was so quiet I could hear the generators reverberating loudly, which light up the dark ground.", Line_Normal);

Reviewer: ```suggestion
			NULL, "The surrounding area was so quiet I could hear the loud reverberations of the generators that lit the dark ground.", Line_Normal);
```

			NULL, " I'm sure Mion will be able to take care of it.", Line_WaitForInput);
	ModPlayVoiceLS(4, 1, "ps3/s14/01/210100968", 256, TRUE);
	OutputLine(NULL, "一人だとろくな考えにならないけど、レナも魅音だったら、色々と話しやすいだろうしさ」",

> 			NULL, "Her isolation would make her opinion so bad, but Mion's there with her so they'll be able to talk things through.\" ", Line_Normal);

Reviewer: ```suggestion
			NULL, " It would be tough to handle on her own, but Mion's there with her so they'll be able to talk things through.\" ", Line_Normal);
```
